README.md

```diff
@@ -2,42 +2,1104 @@
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Async Api
+
+#### Raw Port Async Api V3
+This section illustrates an example of an Async API version 3 configuration for a Trip Status Streaming API. The API is used to stream events related to the Trip entity. It describes the schema for the messages, including the fields and their types.
+```json
+{
+  "mediaType": "text/json",
+  "asyncapi": "3.0.0",
+  "info": {
+    "title": "Trip Status Streaming API",
+    "version": "1.0.0",
+    "description": "This API exposes all events related to `Trip` entity"
+  },
+  "channels": {
+    "transportmng.tripexecution.devents.status": {
+      "messages": {
+        "tripStatusEvent": {
+          "payload": {
+            "schemaFormat": "application/vnd.apache.avro;version=1.9.0",
+            "schema": {
+              "type": "record",
+              "name": "TripStatusChange",
+              "namespace": "com.company-xyz.transportmng.tripexecution",
+              "fields": [
+                {
+                  "name": "id",
+                  "type": {
+                    "avro.java.string": "String",
+                    "type": "string"
+                  }
+                },
+                {
+                  "name": "event_type",
+                  "type": {
+                    "name": "tripEvent",
+                    "symbols": [
+                      "planned",
+                      "booking_started",
+                      "booking_ended",
+                      "loading_started",
+                      "loading_ended",
+                      "departed_from_origin",
+                      "unloading_at_stop_started",
+                      "unloading_at_stop_ended",
+                      "loading_at_stop_started",
+                      "loading_at_stop_ended",
+                      "departed_from_stop",
+                      "arrived_at_destination",
+                      "unloading_started",
+                      "unloading_ended",
+                      "completed"
+                    ],
+                    "type": "enum"
+                  }
+                },
+                {
+                  "name": "event_timestamp",
+                  "type": {
+                    "avro.java.string": "String",
+                    "type": "string"
+                  }
+                },
+                {
+                  "name": "source_system",
+                  "default": "TMS",
+                  "type": {
+                    "avro.java.string": "String",
+                    "type": "string"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+
+#### Raw Port Async Api V2
+This section provides an example of an Async API version 2 configuration for a Trip Status Streaming API. It includes details on the servers, channels, and message schemas used for streaming events related to the Trip entity.
```

Comment:
Non commenti quali risorse vengono create in Blindata a fronte di questo descriptor.